.github/workflows/publish_docker_images_cron.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Clone tools branch
-        run: git clone -b v0.8.18 --depth=1 https://github.com/citusdata/tools.git tools
+        run: git clone -b postgres-15-docker --depth=1 https://github.com/citusdata/tools.git tools
```

Comment:
After closing https://github.com/citusdata/tools/pull/251 task, I will tag tools and will put the latest tag here